src/file/storage/storage.service.ts

```diff
@@ -0,0 +1,17 @@
+import * as express from 'express';
+import * as fs from 'fs';
+import * as mongoose from 'mongoose';
+import { IFile } from '../file.interface';
+import { fileModel } from '../file.model';
+
+export class storageService {
+
+  public static delete(filePath: fs.PathLike) {
+    return fs.unlink(filePath, (err) => { if (err) throw err; });
+  }
+
+  public static update(filePath: String) {
+    // TODO: Change filePath in S3 (fileName) (Necessary?)
```

Comment:
Remember to create an issue for that later

Reply:
V